mobile/src/hooks/use-search.ts

```diff
@@ -84,6 +84,8 @@ export const useSearch = () => {
   }
```

Comment:
ListJsonUsersByPrefix also matches the user's own username. Maybe it should be removed from the search results?